developer-docs-site/docs/concepts/txns-states.md

```diff
@@ -90,7 +90,7 @@ Anyone can submit a transaction to the Aptos blockchain to modify the ledger sta
 
 The Aptos blockchain uses proof to verify the authenticity and correctness of the blockchain data.
 
-All data in the Aptos blockchain is stored in a single-version distributed database. Each validator and fullnode's [storage](./validator-nodes#storage) is responsible for persisting the agreed upon blocks of transactions and their execution results to the database. 
+All data in the Aptos blockchain is stored in a multi-version distributed database. Each validator and fullnode's [storage](./validator-nodes#storage) is responsible for persisting the agreed upon blocks of transactions and their execution results to the database. 
```

Comment:
I think either "single-version" or "multi-version" is confusing in this context. Let's say something along the line of "the same Aptos blockchain data is replicated across the network, to every Aptos node", for example? Some English speaker please advice :trollface: .